pandas/tseries/tools.py

```diff
@@ -224,6 +224,9 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
     infer_datetime_format : boolean, default False
         If no `format` is given, try to infer the format based on the first
         datetime string. Provides a large speed-up in many cases.
+    origin : scalar convertible to Timestamp / `julian`
+        Define relative offset for the returned dates
+        - If `julian`, offset is set to beginning of Julian Calendar
```

Comment:
Can you let the first sentence ('Define ...') end with a `.`, and remove the `-` on this line? As this will not be rendered as a list in this way, but just as continuous text (which is OK here)